meta-resin-jethro/recipes-connectivity/modemmanager/modemmanager_1.6.8.bb

```diff
@@ -0,0 +1,55 @@
+SUMMARY = "ModemManager is a daemon controlling broadband devices/connections"
```

Comment:
typo in commit log: Currently oe provides an older version in fido. This recipe is based on the
one from meta-oe (morty).

Guess that should have been jethro instead of fido

Reply:
Also, doesn't this also need enum-conversion.patch ?